dist/index.js

```diff
@@ -5874,20 +5885,23 @@ async function runAction() {
     return;
   }
 
+  const digest = await docker.getDigest(localTag);
+  if (!digest) {
+    core.error('Unable to retrieve the digest of the image');
```

Comment:
I don't think that not retrieving the `digest` should be a blocker, so we maybe should communicate the error but without stopping the action.